frame/bridge/ethereum/backing/src/lib.rs

```diff
@@ -220,15 +220,6 @@ decl_module! {
 			0
 		}
 
-		fn on_runtime_upgrade() -> frame_support::weights::Weight {
```

Comment:
Remove this will make darwinia has no effect on runtime upgrade?

Reply:
We've 3 runtimes pangolin, crab, darwinia. So I recommend to do a custom runtime upgrade on their runtime.